examples/hello-world-service/hello_world_service.md

```diff
@@ -1,10 +1,14 @@
 # Hello world service
 
-Let's write a simple HTTP service in Ballerina. 
-This example demonstrates the network primitives in the language that make it simpler to develop services.
+Let's write a simple HTTP service in Ballerina. This example demonstrates the network primitives in the language that make it simpler to develop services.
 
 ::: code hello_world_service.bal :::
 
-::: out hello_world_service.client.out :::
+Execute the command below to run the above service.
+
+::: out hello_world_service.server.out :::
 
-::: out hello_world_service.server.out :::
\ No newline at end of file
+In a new CLI tab, execute th cURL command below to invoke the service.
```

Comment:
Can we simplify as `Invoke the service by executing the following cURL command in a new terminal`

Main difference is having the verb at the beginning

Reply:
Updated.